centreon/doc/API/centreon-api-v23.04.yaml

```diff
@@ -4697,7 +4697,7 @@ components:
               type: string
               format: email
```

Comment:
Can you update the "description" section as well ?

Reply:
This modification is only done to make test pass.
Because the login api require an "email" format in the response (with a valid domain pattern), I reflect this in the example.
That's it, nothing more has changed for these lines.